modules/edu.gemini.seqexec.web/edu.gemini.seqexec.web.client/src/main/scala/edu/gemini/seqexec/web/client/semanticui/elements/popup/Popup.scala

```diff
@@ -0,0 +1,39 @@
+// Copyright (c) 2016-2017 Association of Universities for Research in Astronomy, Inc. (AURA)
+// For license information see LICENSE or https://opensource.org/licenses/BSD-3-Clause
+
+package edu.gemini.seqexec.web.client.semanticui.elements.popup
+
+import japgolly.scalajs.react.component.Scala.Unmounted
+import japgolly.scalajs.react.{Callback, ScalaComponent}
+import japgolly.scalajs.react.vdom.html_<^._
+import japgolly.scalajs.react.raw.ReactElement
+
+/**
+  * Produces a popup using javascript
+  * This is a barebones component waiting for the proper react component to be made available
+  */
+@SuppressWarnings(Array("org.wartremover.warts.NonUnitStatements", "org.wartremover.warts.AsInstanceOf"))
+object Popup {
+  @SuppressWarnings(Array("org.wartremover.warts.DefaultArguments"))
+  final case class Props(selector: String, content: String)
+
+  private val component = ScalaComponent.builder[Props]("Popup")
+    .stateless
+    .renderPC{($, p, c) =>
```

Comment:
What is the significance of the `$` here? I see it a few places and I'm unsure what it signifies.

Reply:
the use of `$` is just a convention used in `scalajs-react`. It's probably a bad idea as you get easily confused with `jQuery`'s `$`.

In any case, in this particular call `$` is a reference to the react component which sometimes you need to access. In the most common case you only do `.renderP` and use the `Props` but this is a special case

the `$` reference lets me access the children and render them directly avoiding wrapping them in another markup

Reply:
Okay, that makes sense. Thanks for the explanation! It is much appreciated.